diff --git a/expat/CMakeLists.txt b/expat/CMakeLists.txt
index 1026938..c71e20b 100644
--- a/expat/CMakeLists.txt
+++ b/expat/CMakeLists.txt
@@ -1,9 +1,12 @@
 # This file is copyrighted under the BSD-license for buildsystem files of KDE
 # copyright 2010, Patrick Spendrin <ps_ml@gmx.de>
 
+cmake_minimum_required(VERSION 3.0)
 project(expat)
+include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
+conan_basic_setup()
+
 
-cmake_minimum_required(VERSION 2.6)
 set(PACKAGE_BUGREPORT "expat-bugs@libexpat.org")
 set(PACKAGE_NAME "expat")
 set(PACKAGE_VERSION "2.2.5")
